var/spack/repos/builtin/packages/vecgeom/package.py

```diff
@@ -21,6 +21,11 @@ class Vecgeom(CMakePackage, CudaPackage):
     maintainers("drbenmorgan", "sethrj")
 
     version("master", branch="master")
+    version(
+        "1.2.9",
+        url="https://gitlab.cern.ch/-/project/981/uploads/55a89cafbf48a418bec68be42867d4bf/VecGeom-v1.2.9.tar.gz",
```

Comment:
Different url than other versions. Does the previous schema still work?

Reply:
I've checked and the `https://gitlab.cern.ch/VecGeom/VecGeom/uploads` urls still give a download. I _think_ it's just a change to how GitLab renders these, and the old ones remain active, or forward.